pyseir/inference/model_fitter.py

```diff
@@ -121,6 +137,8 @@ def __init__(
 
         self.fips = fips
         self.ref_date = ref_date
+        self.max_fit_date = (dt.date.today() - timedelta(days=7) - ref_date.date()).days  # natasha
```

Comment:
extract the 7 day time delta into a named constant.

Reply:
Ah, I forgot to delete that. I refactored the code